tendermint/src/time.rs

```diff
@@ -36,13 +38,38 @@ impl Time {
         Ok(Time(DateTime::parse_from_rfc3339(s)?.with_timezone(&Utc)))
     }
 
+    /// Return an RFC 3339 and ISO 8601 date and time string with 6 subseconds digits and Z.
+    pub fn to_rfc3339(&self) -> String {
+        self.0.to_rfc3339_opts(SecondsFormat::Micros, true)
+    }
+
     /// Convert this timestamp to a `SystemTime`
     pub fn to_system_time(&self) -> Result<SystemTime, Error> {
         let duration_since_epoch = self.duration_since(Self::unix_epoch())?;
         Ok(UNIX_EPOCH + duration_since_epoch)
     }
 }
 
+impl Default for Time {
+    fn default() -> Time {
+        Time::unix_epoch()
```

Comment:
I'm not sure a "default time" makes sense

Reply:
Makes sense, just removed.